### PR TITLE
#9 Task. AlexErm

### PR DESCRIPTION
**Pull request by alexErm0**
https://travis-ci.com/github/alexErm0/js-assignments/builds/175557403

### CODE REVIEW

File: task/09-functions-n-closures-tasks.js [hunk 1]
```diff
@@ -171,7 +193,7 @@ function partialUsingArguments(fn) {
  *   getId10() => 11
  */
 function getIdGeneratorFunction(startFrom) {
-    throw new Error('Not implemented');
+    return () => {return startFrom++}
```
Comment (AlexMokin) [inline task/09-functions-n-closures-tasks.js]: braces are not required for simple arrow funcitons 
eg.
`() => value`

**Review by AlexMokin — COMMENTED** [review]
Indentations are not consistent. There are violations of the google's code style.

I'll mark the PR as "Draft", please click "ready for review" when it will be finished. Thank you!

**Comment by AlexMokin** [comment]
Review this https://javascript.info/coding-style . And try to find code styling mistakes yourself and fix them.
**Also theere must not be any changes to the formatting of the original code**

I'll mark the PR as "Draft", please click "ready for review" when it will be finished. Thank you!

**Comment by alexErm0** [comment]
I tried to pull from the master the latest updates, but after that some modules are missing.
image

File: task/09-functions-n-closures-tasks.js [hunk 2]
```diff
@@ -150,7 +166,12 @@ function logger(func, logFunc) {
  *   partialUsingArguments(fn, 'a','b','c','d')() => 'abcd'
  */
 function partialUsingArguments(fn) {
-    throw new Error('Not implemented');
+    let args = Array.from(arguments).slice(1);
+    return function(){
+        let Arg = args.concat(Array.from(arguments));
+        console.log(fn.apply(Arg));
```
Comment (AlexMokin) [inline task/09-functions-n-closures-tasks.js]: why do you need console.log here?

**Review by AlexMokin — COMMENTED** [review]
indentations are still not consistent, there are missing brackets and semicolon.

Merge the latests changes from master

I'll mark the PR as "Draft", please click "ready for review" when it will be finished. Thank you!

**Comment by AlexMokin** [comment]
there are missing brackets and semicolons, find this mistakes and fix them. Also double check indentations. Do it for all PRs

Please do not re-open until fixed

I'll mark the PR as "Draft", please click "ready for review" when it will be finished. Thank you!

**Comment by AlexMokin** [comment]
accepted

**Pull request closed without merging**